src/composeRestyleFunctions.ts

```diff
@@ -45,7 +45,7 @@ const composeRestyleFunctions = <
     },
   ): RNStyle => {
     return funcs.reduce((acc, func) => {
-      return {...acc, ...func(props, {theme, dimensions})};
+      return Object.assign(acc, func(props, {theme, dimensions}));
```

Comment:
Out of curiosity did you notice measurable difference between spread and Object.assign? Or is this more attempting to move away from using spread as much?

Reply:
I believe there was a measurable performance gain for this particular change as well. The difference here is that we're not creating a new object with `Object.assign` on every iteration, as we would have with the spread operator. If this would have been `Object.assign({}, acc, func(props, {theme, dimensions}))` instead, it would likely not have resulted in an improvement.

Reply:
Ah you're right I missed that